test/info.test.mjs

```diff
@@ -37,6 +37,7 @@ describe('aos Info', async () => {
     assert(processInfo.Owner);
     assert(processInfo.Handlers);
     assert.deepStrictEqual(processInfo.Handlers, [
+      '_bootAnt',
```

Comment:
This will go away

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/57/commits/7f04c36b536011cec5286215a0ef1bd830c54ee2